### PR TITLE
Samsung.DataMigration version 14.0.162

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/99332)

### CODE REVIEW

File: manifests/s/Samsung/DataMigration/14.0.162/Samsung.DataMigration.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Samsung.DataMigration
+PackageVersion: 14.0.162
```
Comment (Trenly) [inline manifests/s/Samsung/DataMigration/14.0.162/Samsung.DataMigration.installer.yaml:5]:
```suggestion
PackageVersion: 2.0
```
Writes `2.0` to the control panel when the install works

File: manifests/s/Samsung/DataMigration/14.0.162/Samsung.DataMigration.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Samsung.DataMigration
+PackageVersion: 14.0.162
+PackageLocale: en-US
+Publisher: Macrovision Corporation
```
Comment (Trenly) [inline manifests/s/Samsung/DataMigration/14.0.162/Samsung.DataMigration.locale.en-US.yaml:7]:
```suggestion
Publisher: Samsung
```
Writes `Samsung` to control panel

File: manifests/s/Samsung/DataMigration/14.0.162/Samsung.DataMigration.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Samsung.DataMigration
+PackageVersion: 14.0.162
+PackageLocale: en-US
+Publisher: Macrovision Corporation
+PackageName: Samsung Data Migration Tool for SSDs
```
Comment (Trenly) [inline manifests/s/Samsung/DataMigration/14.0.162/Samsung.DataMigration.locale.en-US.yaml:8]:
```suggestion
PackageName: Samsung Data Migration
```

File: manifests/s/Samsung/DataMigration/14.0.162/Samsung.DataMigration.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Samsung.DataMigration
+PackageVersion: 14.0.162
```
Comment (Trenly) [inline manifests/s/Samsung/DataMigration/14.0.162/Samsung.DataMigration.yaml:5]:
```suggestion
PackageVersion: 2.0
```
Writes `2.0` to the control panel when the install works

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by upintheairsheep** [comment]
Re-open please

**Pull request closed without merging**